libs/execution_context/ebpf_core.c

```diff
@@ -2779,6 +2800,49 @@ _ebpf_core_memmove_s(
     return memmove_s(destination, destination_length, source, source_length);
 }
 
+
+_IRQL_requires_max_(DISPATCH_LEVEL)
+static uint64_t
+_ebpf_core_get_current_process_start_key(
+    uint64_t dummy_param1,
+    uint64_t dummy_param2,
+    uint64_t dummy_param3,
+    uint64_t dummy_param4,
+    uint64_t dummy_param5,
+    _In_ const bpf_sock_ops_t* ctx)
+{
+    UNREFERENCED_PARAMETER(dummy_param1);
+    UNREFERENCED_PARAMETER(dummy_param2);
+    UNREFERENCED_PARAMETER(dummy_param3);
+    UNREFERENCED_PARAMETER(dummy_param4);
+    UNREFERENCED_PARAMETER(dummy_param5);
+    UNREFERENCED_PARAMETER(ctx);
+    if (KeGetCurrentIrql() > DISPATCH_LEVEL)
```

Comment:
I think we dont need check for IRQL > DISPATCH_LEVEL

Reply:
resolving per offline discussion.